Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift

```diff
@@ -104,6 +109,87 @@ final class XcbeautifyLibTests: XCTestCase {
     }
 
     func testExecuted() {
+        let input1 = "Test Suite 'All tests' failed at 2022-01-15 21:31:49.073."
+        let input2 = "Executed 3 tests, with 2 failures (1 unexpected) in 0.112 (0.112) seconds"
+        
+        let input3 = "Test Suite 'All tests' passed at 2022-01-15 21:33:49.073."
+        let input4 = "Executed 1 test, with 1 failure (1 unexpected) in 0.200 (0.200) seconds"
+        
+        // First test plan
+        XCTAssertNil(parser.summary)
+        XCTAssertFalse(parser.needToRecordSummary)
+        let formatted1 = noColoredFormatted(input1)
+        XCTAssertTrue(parser.needToRecordSummary)
+        let formatted2 = noColoredFormatted(input2)
+        XCTAssertFalse(parser.needToRecordSummary)
+        XCTAssertNil(formatted1)
+        XCTAssertNil(formatted2)
+        XCTAssertNotNil(parser.summary)
```

Comment:
I would unwrap it here to simplify subsequent code
```suggestion
        let summary = try XCTUnwrap(parser.summary)
```